Client/N3Base/N3Camera.cpp

```diff
@@ -60,6 +60,7 @@ void CN3Camera::Release()
 	CN3Transform::Release();
 }
 
+// NOTE(Gilad): I don't think we use camera files, do we? as the load function never gets executed.
```

Comment:
They will get used, ultimately. I know for a fact they're used by the 1.298 character selection screen, at the very least.